### PR TITLE
Adding snips-injection

### PR DESCRIPTION
**Pull request by s00500**
#525
- Update dockerfile
- Snips injection version print
- Start snips-injection in run
- Versionbump to 5.2

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @s00500, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#546).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by homeassistant** [comment]
Hi @s00500, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;hassio-addons#546).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by s00500** [comment]
Changes added, sry for the weird char in the jsonfile

File: snips/Dockerfile [hunk 1]
```diff
@@ -13,16 +13,16 @@ RUN apt-get update \
...
-            bash -c 'echo "deb https://raspbian.snips.ai/stretch stable main" > /etc/apt/sources.list.d/snips.list' \
-            && apt-key adv --keyserver pgp.surfnet.nl --recv-keys D4F50CDCA10A2849; \
-        fi
+    then \
+    bash -c 'echo "deb https://debian.snips.ai/stretch stable main" > /etc/apt/sources.list.d/snips.list' \
+    && apt-key adv --keyserver pgp.surfnet.nl --recv-keys F727C778CCB0A455; \
+    else \
+    bash -c 'echo "deb https://raspbian.snips.ai/stretch stable main" > /etc/apt/sources.list.d/snips.list' \
+    && apt-key adv --keyserver pgp.surfnet.nl --recv-keys D4F50CDCA10A2849; \
+    fi
```
Comment (pvizeli) [inline snips/Dockerfile]: I don't see a real change here, please revert this part

Comment (s00500) [reply snips/Dockerfile]: done

**Pull request merged**